### ISSUE TITLE
generated HTML contains invalid anchor links

### ISSUE THREAD

**Issue by yaccz** [swagger-api/swagger-codegen#5087]
##### Description

generated HTML contains invalid anchor links on types of request parameters

##### Swagger-codegen version

current master; HEAD = 1c160df33897898da1c39f230c737eb6a313e6ba

##### Swagger declaration file content or url

```yaml
swagger: "2.0"

info:
  title: swagger-codegen broken anchors to models
  version: '1.0'

definitions:
  foo:
    type: object
    properties:
      x:
        type: string

paths:
  /foo:
    get:
      parameters:
        - name: foo
          in: body
          description: foo
          schema: {$ref: "#/definitions/foo"}
      responses:
        "200":
          description: foo
          type: null
```

Note the links first letter is capital while the anchor is not.

##### Command line used for generation

JAR=".../swagger-codegen.git/modules/swagger-codegen-cli/target/swagger-codegen-cli.jar"
java -jar "${JAR}"  generate -i ~/sscce.yaml -l html -o foo

**Comment by wing328** [comment]
@yaccz thanks for reporting the issue. May I know if you've time to contribute a fix? If yes, I can show you some good starting points.

`html2` is another documentation generator worth trying.

**Comment by yaccz** [comment]
I'm switching to html2. Maybe the documentation [1]_ should be updated. Thanks.

.. [1] https://github.com/swagger-api/swagger-codegen#generating-static-html-api-documentation

**Comment by wing328** [comment]
@yaccz yup, please feel free to send a PR to update the README about `html2`

**Comment by yaccz** [comment]
Is there a way to display all possible target formats? I couldn't find it using 
```
JAR=".../swagger-codegen.git/modules/swagger-codegen-cli/target/swagger-codegen-cli.jar"
java -jar "${JAR}"  help generate
```

or how can I get that information?

**Comment by wing328** [comment]
Please run `java -jar modules/swagger-codegen-cli/target/swagger-codegen-cli.jar config-help -l xyz ` it will show an error and display a list of possible targets.
